app/src/components/InformationAboutMe.css

```diff
@@ -0,0 +1,47 @@
+.photo_container {
+  width: 130px;
+  height: 130px;
+}
+
+.photoAvatar {
+  height: 100%;
+  width: 100%;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+}
+
+.article-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h2 {
+  color: var(--yellow);
+  font-size: 14px;
+  font-weight: 500;
+  letter-spacing: 0.2px;
+  text-align: center;
+  align-items: center;
+  margin: 0;
+  font-family: "Open Sans";
```

Comment:
it is better to use a spare font is this one is not loaded for some reasons

`font-family: "Open Sans", serif;`